crates/exex/src/model.rs

```diff
@@ -53,6 +54,31 @@ impl KethMaybeRelocatable {
     pub fn one() -> Self {
         Self(Felt252::ONE.into())
     }
+
+    /// Creates a [`KethMaybeRelocatable`] instance from a byte slice in big-endian order.
+    ///
+    /// This method takes a byte slice and converts it into a [`KethMaybeRelocatable`]
+    /// by interpreting the bytes as a [`Felt252`] value in big-endian format.
+    ///
+    /// # Parameters
+    ///
+    /// - `bytes`: A slice of bytes that represents a big-endian encoded value.
+    ///
+    /// # Returns
+    ///
+    /// Returns an instance of [`KethMaybeRelocatable`] that corresponds to the given byte slice.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use kakarot_exex::model::KethMaybeRelocatable;
+    ///
+    /// let bytes: &[u8] = &[0x00, 0x01, 0x02, 0x03]; // Example big-endian byte array
+    /// let keth_value = KethMaybeRelocatable::from_bytes_be_slice(bytes);
+    /// ```
+    pub fn from_bytes_be_slice(bytes: &[u8]) -> Self {
+        Felt252::from_bytes_be_slice(bytes).into()
+    }
```

Comment:
I'm not sure that we actually need this, this would mean that we have data in blocks/state that is a `[u8]` and that we want to data to be loaded in Cairo as a `felt`. I can't see any use case for this (apart from using this together with `u64 to bytes_be for example, but in this latter context it may be better to directly implement such a method

Reply:
This is directly used by the `From<B256> for KethU256` conversion. Indeed, here we take bytes in big endian order and transform them into `MaybeRelocatable` via this helper, that is to say `KethMaybeRelocatable::from_bytes_be_slice(&value.0[16..])`. This is just an helper.